v2/thirdpartyemailpassword/migration/bulk-user-import-schema-examples.mdx

```diff
@@ -0,0 +1,200 @@
+              },
+              "required": ["secret"]
+            }
+          }
+        },
+        "loginMethods": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "recipeId": {
+                "type": "string",
+                "description": "The recipe id for this login method.",
+                "enum": ["emailpassword", "thirdparty", "passwordless"]
+              },
+              "tenantIds": {
+                "type": "array",
+                "items": {
+                  "type": "string",
+                  "description": "The tenantId of the tenant where this login method should be created."
+                },
+                "default": ["public"]
+              },
+              "email": {
+                "type": "string",
+                "description": "The email of the user. This is applicable for all the recipes but only required in emailpassword and thirdparty recipe.",
+              },
+              "passwordHash": {
+                "type": "string",
+                "description": "Hashed password for the user. This is required for the emailpassword recipe.",
+              },
+              "hashingAlgorithm": {
+                "type": "string",
+                "description": "The hashing algorithm used to hash the password. This is required for the emailpassword recipe.",
+                "enum": ["bcrypt", "argon2", "firebase_scrypt"],
+              },
+              "isVerified": {
+                "type": "boolean",
+                "description": "Whether the email is verified or not.",
+                "default": false
+              },
+              "isPrimary": {
+                "type": "boolean",
+                "description": "Whether this login method is the primary login method for the user. Only one login method can be primary.",
+                "default": false
+              },
+              "timeJoinedInMSSinceEpoch": {
+                "type": "integer",
+                "description": "The time the user joined in milliseconds since epoch.",
+              },
+              "thirdPartyId": {
+                "type": "string",
+                "description": "The id of the third party provider. This is required for the thirdparty recipe.",
+              },
+              "thirdPartyUserId": {
+                "type": "string",
+                "description": "The id of the user in the third party provider. This is required for the thirdparty recipe.",
+              },
+              "phoneNumber": {
+                "type": "string",
+                "description": "The phone number of the user. This is applicable for the passwordless recipe. Required if email is not passed.",
+              },
+              "required": ["recipeId"]
+            }
+          }
+        },
+        }
+      },
+      "required": ["loginMethods"]
+    }
+  }
+
+```
+
+## Examples
+
+### Example 1: Sample Bulk User Import JSON
```

Comment:
I think you must provide many more examples. All in one examples might not be very effective. It should be a step by step additions.

For example:
Start with a very simple use case, of importing emailpassword users only. then, thirdparty users.
Then probably add an example on how to assign user roles. Later metadata.
Then give examples of account linking where same user has different login methods. Talk about isPrimary. Talk about where to add user roles and metadata.

Think about this in more detail please.